packages/teams-js/src/public/media.ts

```diff
@@ -446,7 +474,7 @@ export namespace media {
    */
   abstract class MediaController<T> {
     /** Callback that can be registered to handle events related to the playback and control of video content. */
-    protected controllerCallback: T;
+    protected controllerCallback?: T;
```

Comment:
>? [](http://example.com/codeflow?start=32&length=1)

Despite not being declared this way previously, this value could always have been `undefined` as it is only set in the constructor (immediately below this) and the constructor clearly accepts `undefined`